File: II sem/Programowanie Obiektowe/Projekty/projekt2/po3/src/Wilk.java
public class Wilk extends Zwierze {
    public Wilk(String ikonka, int x_pos, int y_pos, int sila, int inicjatywa, Swiat swiat)
    {
        super(ikonka, x_pos, y_pos, sila, inicjatywa, swiat);
    }

    public Wilk(Organizm organizm)
    {
        super(organizm);
    }

    @Override
    public Organizm kopiuj(Organizm organizm) {
        return new Wilk(organizm);
    }

}

File: II sem/Programowanie Obiektowe/Projekty/projekt1/POproject1/Wilk.cpp
#include "Wilk.h"

Wilk::Wilk(int x, int y) : Zwierze(EMOJI ? u8"Wi" : "Wi", 9, 5, x, y)
{

}

Wilk* Wilk::potomek()
{
	return new Wilk(*this);
}